.jest.json

```diff
@@ -1,7 +1,5 @@
 {
-  "collectCoverage": false,
   "preset": "react-native",
-  "verbose": false,
```

Comment:
what happend here?

Reply:
by default, it is set as `false`

https://github.com/facebook/jest/blob/master/packages/jest-cli/src/lib/update_global_config.js#L120-#L122